Makefile

```diff
@@ -18,4 +18,13 @@ init:
 tendermint-start:
 	tendermint start --home ~/.gaia-rs
 
-.PHONY: run run-debug test install init tendermint-start
\ No newline at end of file
+init-second:
```

Comment:
Second?

Reply:
We'll probably move this stuff into Docker containers soon.